src/utils/isReactCreateClassCall.js

```diff
@@ -32,3 +32,35 @@ export default function isReactCreateClassCall(path: NodePath): boolean {
   var module = resolveToModule(path.get('callee', 'object'));
   return Boolean(module && isReactModuleName(module));
 }
+
+
+/**
+ * Returns true if the expression is a function call of the form
+ * ```
+ * import createReactClass from 'create-react-class';
+ * createReactClass(...);
+ * ```
+ */
+function isReactCreateClassCallModular(path: NodePath): boolean {
+  if (types.ExpressionStatement.check(path.node)) {
+    path = path.get('expression');
+  }
+
+  if (!match(path.node, {type: 'CallExpression'})) {
+    return false;
+  }
+  var module = resolveToModule(path.get('callee', 'object'));
```

Comment:
This might actually be the culprit. The value of `callee` for the `CallExpression` is `Identifier`, which doesn't have an `object` field: http://astexplorer.net/#/gist/2830b1b852ac298dab2f18802133afdf/ec06e05ba93e93903bb7c85a1bd6a12622d41918 .

I think you can do `resolveToModule(path)` here, that should solve the issue.